docs/source/checks/vision/model_evaluation/plot_single_dataset_performance.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+"""
+.. _plot_vision_single_dataset_performance:
+
+Single Dataset Performance
+*********************************
+
+This notebooks provides an overview for using and understanding single dataset performance check.
+
+**Structure:**
+
+* `What Is the Purpose of the Check? <#what-is-the-purpose-of-the-check>`__
+* `Generate data an model <#generate-data-and-model>`__
+* `Run the check <#run-the-check>`__
+* `Define a condition <#define-a-condition>`__
+
+What Is the Purpose of the Check?
+=================================
+This check returns the results from a dict of metrics, in the format metric name: ignite.Metric, calculated for the
+given model dataset.
+
+"""
+#%%
+# Generate data and model
+# -----------------------
+
+from deepchecks.vision.checks import SingleDatasetPerformance
+from deepchecks.vision.datasets.classification import mnist
+
+#%%
+
+
+mnist_model = mnist.load_model()
+train_ds = mnist.load_dataset(train=True, object_type='VisionData')
+
+#%%
+# Run the check
+# -------------
+# We will run the check with the model defined above.
+#
+# The check will use the default classification metrics - `Precision
```

Comment:
Would add here information on the scorers argument: 
"You can select which scorers to use by passing either a list or a dict of scorers to the check,
# the full list of possible scorers can be seen at ..."

Reply:
@Nadav-Barak We'll probably add a link to metrics guide to all the relevant plots after it's complete

Reply:
Would still add this sentence, for now link to metrics.py then afterward we could change the link to the metric guide

Reply:
metrics.py is internal so we shouldn't link to the user